src/js/edu-benefits/reducers/uiState/index.js

```diff
@@ -21,83 +20,67 @@ const ui = {
   },
   pages: {
     '/introduction': {
-      complete: false,
-      verified: false,
+      reviewEdit: false,
```

Comment:
The name of this property is a little confusing to me. To me on the review page there are 2 states: `review` or `edit`. I think I know what you're trying to get at here, is the state in `edit` on the review page? Maybe call this `editOnReview` on something like that instead?